### ISSUE TITLE
Custom Mob Spawner file in Mo Creatures downloads

### ISSUE THREAD

**Issue by DWX12** [DrZhark/mocreatures#102]
The Custom Mob Spawner file got uploaded to the Mo Creatures page. Figured it's probably just a mistake that happened.

**Comment by Excaliburning** [comment]
Noticed this too.
And the Mo Creature 1.12.2 file is missing instead.

**Comment by MokahTGS** [comment]
The Mo Creatures 1.12.2 file is there under the files section, just not listed on the front page of the Curse entry.

Custom Mob Spawner should be listed under it's own curse entry and then flagged as a requirement for Mo Creatures.

**Comment by Excaliburning** [comment]
Yup, alright. It got uploaded at the same time as a sub-file, my bad.

**Comment by CrankySupertoonArchive** [comment]
Its important because us modpack dev cannot use two versions of the same mod in a pack at one times. Having both files under the same project can be a issue when using the Twitch Curse App

**Comment by ghost** [comment]
Actually they are separate projects, someone just bungled up somewhere and uploaded the files to the wrong place. Furthermore the version Mo' creatures 12.0.5 downloads is for 1.10.

**Comment by Shabnakh-adyr** [comment]
Just wanted to chime in this still an issue. It's annoying outside of modpacks because the Twitch app will "update" the mod to Custom Mob Spawner.. and then download the old Mob Spawner as a dependency.
